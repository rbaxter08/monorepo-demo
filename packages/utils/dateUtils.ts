import { addDays as _addDays } from "date-fns";

export function addDays(date: number | Date, amount: number) {
  return _addDays(date, amount);
}
