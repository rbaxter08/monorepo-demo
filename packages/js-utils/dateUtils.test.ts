import { addDays } from "./dateUtils";

describe("dateUtils", () => {
  it("addDays() should add days", () => {
    const result = addDays(new Date("2020-04-13T00:00:00.000+08:00"), 3);
    expect(result).toMatchSnapshot();
  });
});
