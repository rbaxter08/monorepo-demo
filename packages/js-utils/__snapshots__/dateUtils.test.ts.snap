// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dateUtils addDays() should add days 1`] = `2020-04-15T16:00:00.000Z`;
